features/step_definitions/cli.rb

```diff
@@ -32,6 +32,21 @@
   ensure_admin_tagged
   opts = table.raw == [["dummy"]] ? [] : opts_array_process(table.raw)
 
+  if yaml_key == "oadm_policy_add_role_to_user"
+    teardown_add {
+      yaml_del_key="oadm_policy_remove_role_from_user"
+      if background
+        @result = env.admin.cli_exec(
+          yaml_del_key.to_sym,
+          opts << [ :_background, true ]
```

Comment:
I don't think clean-up will need to happen in the background. Also I don't understand why we would need clean-up at all. Because projects are usually removed after test execution anyway.